app/assets/javascripts/components/common/OverviewStats/namespace_overview_stats.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import OverviewStat from './overview_stat';
+
+const NamespaceOverviewStats = ({ statistics }) => {
+  return (
+    <div className="stat-display">
+      <OverviewStat
+        id="articles-created"
+        className="stat-display__value"
+        stat={statistics.new_count}
+        statMsg={I18n.t('metrics.articles_created')}
+        renderZero={false}
+      />
+      <OverviewStat
+        id="articles-edited"
+        className="stat-display__value"
+        stat = {statistics.edited_count}
+        statMsg={I18n.t('metrics.articles_edited')}
+        renderZero={true}
+      />
+      <OverviewStat
+        id="total-edits"
+        className="stat-display__value"
+        stat = {statistics.revision_count}
+        statMsg={I18n.t('metrics.edit_count_description')}
+        renderZero={true}
+      />
+      <OverviewStat
+        id="student-editors"
```

Comment:
I don't think the user count needs to be in any namespace-specific stats.

Reply:
Won't it give insight of how many students worked on particular tracked namespace?